virttest/libvirt_vm.py

```diff
@@ -937,10 +937,10 @@ def add_nic(help_text, nic_params):
                 if (
                     nettype
                     and nic_queues
-                    and has_sub_option("network", "driver_queues")
+                    and has_sub_option("network", r"driver[._]queues")
```

Comment:
```suggestion
                    and has_sub_option("network", "driver[._]queues")
```
I think that we do not need to pass the Python’s raw string notation for regular expression patterns since there already been: https://github.com/avocado-framework/avocado-vt/blob/92402c95d97d4bbc0686fcc32b5efd5857d4c8ed/virttest/libvirt_vm.py#L465

Thanks.